### PR TITLE
docs: Updates for Indexer 3.0.0

### PR DESCRIPTION
**Pull request by winder**
## Summary

Update documentation to reflect Indexer 3.0.0. References to Indexer writers have been removed or replaced with references to Conduit.

### CODE REVIEW

**Review by algoanne — APPROVED** [review]
Only thing we should probably add is the note to guide people who used to be on 2.x to the migration guide. As is, they'll come here and be confused.

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 # Algorand Indexer
 
-The Indexer is a standalone service that reads committed blocks from the Algorand blockchain and maintains a database of transactions and accounts that are searchable and indexed.
+The Indexer is an API that provides search capabilities to Algorand on-chain data. Data is written by [Conduit](https://github.com/algorand/conduit/blob/master/docs/tutorials/IndexerWriter.md) to populate a PostgreSQL compaatible database.
```
[the remark this answers is not in the document]
Comment (winder) [reply README.md:16]:
```suggestion
The Indexer is an API that provides search capabilities for Algorand on-chain data. Data is written by [Conduit](https://github.com/algorand/conduit/blob/master/docs/tutorials/IndexerWriter.md) to populate a PostgreSQL compatible database.
```

File: README.md [hunk 2]
```diff
@@ -322,13 +282,3 @@ When searching by an account, results are returned most recent first. The intent
 For all other transaction queries, results are returned oldest first. This is because it is the physical order they would normally be written in, so it is going to be faster.
 
 <!-- USAGE_END_MARKER_LINE -->
-
-# Migrating from Indexer v1
```
[the remark this answers is not in the document]
Comment (winder) [reply README.md:326]: Added something about this.

**Review by Eric-Warehime — COMMENTED** [review]
`+16 −1,325` 🔥

**Pull request merged**